### DISCUSSION TITLE
Issue with ChunkedHTML not working as expected

### DISCUSSION THREAD

**Discussion by myfta** [jgm/pandoc#10229] [General]
I am trying to use the ChunkedHTML feature to parse a HTML file into a set of individual files splitting on the level 1 header `<h1>`

Here is my Pandoc command:

`pandoc -t chunkedhtml --split-level=1 --template=chunked-template.html -o RN_File ResTest1.html`

The issue appears to be that the HTML file has a `<div>` wrapper at the beginning of the `<body>` which is treated as level 1 and any subsequent `<h1>` tags are ignored. How can I ignore the `<div>` tag, apart from manually editing the HTML file before I run Pandoc?

Here is the full file content:

```
<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
</head>
<body>
<div class="WRAPPER_Content">    <!-- this appears to the problem line -->

<h1 class="Hdg1">First Name 1</h1>
<div class="fhsection fhsecdata">
<p>Title: Part 1</p>
<p>Type: Task</p>
<p>Date:</p>
<p>Description: M Type</p>
<p>Status: +hold</p>
<p>Repository:</p>
<p>Last action: 5 May 2024</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
</div>

<h1 class="Hdg1">First Name 2</h1>
<div class="fhsection fhsecdata">
<p>Title: Part 2</p>
<p>Type: Task</p>
<p>Date:</p>
<p>Description: D Type</p>
<p>Status: +hold</p>
<p>Repository:</p>
<p>Last action: 5 May 2024</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
</div>

<h1 class="Hdg1">First Name 3</h1>
<div class="fhsection fhsecdata">
<p>Title: Part 3</p>
<p>Type: Task</p>
<p>Date:</p>
<p>Description: B Type</p>
<p>Status: +hold</p>
<p>Repository:</p>
<p>Last action: 5 May 2024</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
</div>

</div>
</body>
</html>

```

**Comment by jgm** [comment]
You could use a filter for this.
Something like (untested):
```lua
function Div(el)
  if #el.classes > 0 and el.classes[1] == 'WRAPPER_Content' then
    return el.content
  end
end
```

**Reply by myfta** [reply]
In case anyone else has a similar issue. I had to work out how to use this function. Turns out it is a LUA filter. So I saved it as remWrap.lua, then ran the Pandoc command as:

`pandoc -t chunkedhtml --lua-filter=remWrap.lua --split-level=1 --template=chunked-template.html -o RN_File2 ResTest1.html`
